scalafmt-core/shared/src/main/scala/org/scalafmt/config/ProjectFiles.scala

```diff
@@ -7,7 +7,7 @@ import org.scalafmt.util.OsSpecific
 case class ProjectFiles(
     git: Boolean = false,
     files: Seq[String] = Nil,
-    includeFilters: Seq[String] = Seq(".*\\.scala", ".*\\.sbt"),
+    includeFilters: Seq[String] = Seq(".*\\.scala$", ".*\\.sbt$"),
```

Comment:
This was, I think, a bug. Currently it will pick up, for example, `something.scalatex` files.